### PR TITLE
OpenRefine lesson updates

### PR DESCRIPTION
**Pull request by wetneb**
Hello, thank you so much for maintaining this fantastic OpenRefine lesson, which is still a reference.

I went through the tutorial and fixed a few inconsistencies I found, mostly updating for changes that we have made in OpenRefine itself.

I haven't touched the Spanish and Portuguese translations because I don't master those languages but I could update the links already if that's welcome.

### Checklist

- [ ] Assign yourself in the "Assignees" menu
- [ ] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [ ] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Manager @anisa-hawes 
- [ ] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [ ] Assign at least one individual or team to "Reviewers"
- [ ] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.

### CODE REVIEW

File: en/lessons/cleaning-data-with-openrefine.md [hunk 1]
```diff
@@ -153,8 +153,8 @@ downloaded data file and click **Next**. By default, the first line will
 be correctly parsed as the name of a column, but you need to unselect
 the 'Quotation marks are used to enclose cells containing column
 separators' checkbox, since the quotes inside the file do not have any
-meaning to *OpenRefine*. Additionally, select the 'Parse cell text into
-numbers, dates, ...' checkbox to let OpenRefine automatically detect
+meaning to *OpenRefine*. Additionally, select the 'Attempt to parse cell
+text into numbers' checkbox to let OpenRefine automatically detect
```
Comment (wetneb) [inline en/lessons/cleaning-data-with-openrefine.md:157]: This reflects an update that we made in the tool, changing the name of this option to match its actual behavior (it doesn't actually parse dates).

File: en/lessons/cleaning-data-with-openrefine.md [hunk 2]
```diff
@@ -292,7 +292,7 @@ cells**, **Join multi-valued cells**, **OK**. Choose the pipe character (|) as a
 separator. The rows now look like before, with a multi-valued Categories
 field.
 
-### Applying ad-hoc transformations through the use of regular expressions
+### Applying ad-hoc transformations through the use of GREL expressions
```
Comment (wetneb) [inline en/lessons/cleaning-data-with-openrefine.md:295]: This section does not use regular expressions at all, only GREL expressions, so I think it's worth avoiding this confusion.

File: en/lessons/cleaning-data-with-openrefine.md [hunk 3]
```diff
@@ -379,7 +379,7 @@ Once your data has been cleaned, you can take the next step and explore
 other exciting features of *OpenRefine*. The user community of
 *OpenRefine* has developed two particularly interesting extensions which
 allow you to link your data to data that has already been published on
-the Web. The [RDF Refine extension][] transforms plaintext keywords into
+the Web. The [RDF Transform extension][] transforms plaintext keywords into
```
Comment (wetneb) [inline en/lessons/cleaning-data-with-openrefine.md:382]: The RDF Transform is the successor of this extension. It should work with recent versions of OpenRefine.

**Comment by anisa-hawes** [comment]
Hello Antonin Delpeuch @wetneb. 

Many thanks for making contact, and offering to support an update of the lesson cleaning-data-with-openrefine and its subsequent translations. 

We actually prefer outside contributors not to create direct Pull Requests from their forks of our repo. Updates to our site are so regular that knots in forks are very frequent.

I'll transfer this to an Issue, create a branch on our side and replicate the changes you suggest. I'll then seek the support of our ES and PT teams to update their translations, and ask our Managing Editors to review and approve these changes.

Thank you for sharing your expertise ✨

**Comment by anisa-hawes** [comment]
The updates advised by @wetneb are now being prepared in #3216.

Thank you again, Antonin.

**Pull request closed without merging**